### PR TITLE
Add ZEC, BCH, and LTC to BitGO wallet plugin

### PR DESCRIPTION
**Pull request by RafaelTaranto**
Milestone: 7.3.x Crafty Chnemu

### CODE REVIEW

File: lib/plugins/wallet/bitgo/bitgo.js [hunk 1]
```diff
@@ -8,35 +9,43 @@ const pjson = require('../../../../package.json')
...
 }
 
-function getWallet (account) {
+function getWallet (account, cryptoCode) {
   const bitgo = buildBitgo(account)
-  const coin = account.environment === 'test' ? 'tbtc' : 'btc'
+  const coin = account.environment === 'test' ? `t${cryptoCode.toLowerCase()}` : cryptoCode.toLowerCase()
 
-  return bitgo.coin(coin).wallets().get({ id: account.walletId })
+  return bitgo.coin(coin).wallets().get({ id: account[`${cryptoCode}WalletId`] })
```
Comment (RafaelTaranto) [inline lib/plugins/wallet/bitgo/bitgo.js]: This feels awkward, thoughts on how to improve it?

File: lib/plugins/wallet/bitgo/bitgo.js [hunk 2]
```diff
@@ -8,35 +9,43 @@ const pjson = require('../../../../package.json')
...
   return checkCryptoCode(cryptoCode)
-    .then(() => getWallet(account))
+    .then(() => getWallet(account, cryptoCode))
     .then(wallet => {
       const params = {
-        address: address,
+        address: getConvertedAddress(address, cryptoCode),
         amount: cryptoAtoms.toNumber(),
-        walletPassphrase: account.walletPassphrase
+        walletPassphrase: account[`${cryptoCode}WalletPassphrase`]
```
Comment (RafaelTaranto) [inline lib/plugins/wallet/bitgo/bitgo.js]: Same as above.

**Comment by RafaelTaranto** [comment]
[Expanded BitGo support - ZEC, BCH, LTC](https://trello.com/c/B9hwfQmc/411-expanded-bitgo-support-zec-bch-ltc)

File: lib/plugins/wallet/bitgo/bitgo.js (hunk 1 above)
Comment (joshmh) [reply lib/plugins/wallet/bitgo/bitgo.js:23]: I think this is good for now.

**Pull request merged**